app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    commands = command.split(" ")
```

Comment:
by default `split()` function split by space, so no reason to write it 
```suggestion
    commands = command.split()
```